File: app/src/main/java/com/neome/java/api/ent/ent/msg/MsgEntSpreadsheetState.java
// neome.ai API - do not change
//

package com.neome.java.api.ent.ent.msg;

import com.neome.java.api.meta.base.Types.MetaIdSpreadsheet;
import com.neome.java.api.nucleus.base.msg.Msg;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class MsgEntSpreadsheetState extends Msg
{
  @Nullable
  public String gridVer;

  public MetaIdSpreadsheet spreadsheetId;
}

File: app/src/main/java/com/neome/api/ent/ent/msg/MsgEntSpreadsheetState.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.ent.msg

import com.neome.api.meta.base.Types.MetaIdSpreadsheet
import com.neome.api.nucleus.base.msg.Msg

interface MsgEntSpreadsheetState : Msg
{
  val gridVer: String?
  val spreadsheetId: MetaIdSpreadsheet
}